src/SourceBuild/Arcade/tools/TextOnlyPackages.csproj

```diff
@@ -0,0 +1,43 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>net6.0</TargetFrameworks>
+    <RestorePackagesPath>$(TargetPackagesPath)</RestorePackagesPath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageDownload Include="Microsoft.DotNet.Common.ItemTemplates"
+                    Version="[$(MicrosoftDotNetCommonItemTemplates21PackageVersion)];
```

Comment:
Nit - The alignment seems off here as well as for ItemTemplates

Reply:
Fixed